### PR TITLE
Refactor: add style

### PR DESCRIPTION
**Pull request by krisbobo**
## Milestone: Style
*The following changes are being introduced:*
- Updated style of the app
- Progress bar with an arbitrary implementation of a counter.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -33,23 +41,46 @@ nav a {
...
+
+.item-container {
+  display: flex;
+  align-items: center;
+  background-color: white;
+  padding: 0.8em;
+  margin-bottom: 1.3em;
+  border: 1px solid rgb(220, 220, 220, 0.7);
+  border-radius: 0.3em;
+}
```
Comment (brainconnect93) [inline src/index.css]: - [x] I will kindly suggest you work on the alignment of the books and the form as seen in the original design.
- [x] Also in the original design there is a profile icon at the top right hand side of the page, currently the profile icon is missing in your design. Kindly check out where i have highlighted in the image below to make correction.


| Your Design 👇 | Zeplin Design 👇 |
| ------------- | ------------- |
| book| Bookstore CMS|

File: src/index.css [hunk 2]
```diff
@@ -131,3 +161,57 @@ nav a {
...
+  padding-left: 1em;
+}
+
+.percent {
+  font-size: 1.8em;
+  font-weight: 400;
+}
+
+.progress {
+  display: flex;
```
Comment (brainconnect93) [inline src/index.css]: - [x] Hello @krisbobo, I notice repetition of `display: flex;` in all your container, One golden tip in styling is that we can create a class with any styling that is repeating and then give that class to any element we want. for example I can write
```CSS
.flex {
display: flex;
}
```
and then give `flex` class to any container I want. you can apply this rule to save a lot of lines in your CSS.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @krisbobo, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️  
- Styling of the bookstore CMS project ✔️ 
- Zeplin Design template used for the design ✔️ 
- There is good use of Git flow ✔️ 

### Changes Required

- [x] Kindly check the link in your `live demo` heading of your README file as it is currently throwing an `error 404`. kindly replace the link with a `text` saying `Not Available` or remove the live demo heading since you don't have a deployed link yet.

Live Demo 👇 

image

- [x] Kindly work on the MIT license link as well as it is current throwing an error

MIT 👇 

image

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (krisbobo) [reply src/index.css]: Hi  @brainconnect93 
Sweet in the middle😅
Duly noted👍

**Review by binhussen — APPROVED** [review]
Hi @krisbobo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**